### DISCUSSION TITLE
以前用官方JAVE，问题贼多，想换其它JAVE，有什么推荐吗？

### DISCUSSION THREAD

**Discussion by TXSQ2009** [Meloong-Git/PCL#1595] [提问] [answered]
以前用官方JAVE，问题贼多，想换其它JAVE，有什么推荐吗？

**Comment by hejiehao** [comment]
bellsoft

**Reply by TXSQ2009** [reply]
谢谢

**Comment by Pigeon0v0** [comment]
Zulu

**Reply by TXSQ2009** [reply]
谢谢

**Comment by WorldHim** [comment, marked as answer]
优化较好的如lunar端用的azul zulu，还有阿里的dragonwell；其他如openjdk。
mcbbs上面有对各java优化的比较：
[[2022.1.29] 换个 JVM，最高节省43.3%内存，提高36.9%CPU效率。](https://www.mcbbs.net/thread-1232993-1-1.html)

**Reply by TXSQ2009** [reply]
谢谢

**Comment by MoYuan-CN** [comment]
提醒:是 **Java** 而不是 Jave

**Reply by Chiloven945** [reply]
应该看得出来是整活吧（

**Reply by TXSQ2009** [reply]
我知道，只不过刚玩mc的时候，以为读音所以一直打出jave，后来知道是java后，习惯已经改不了了= ）

**Reply by SqrtTan** [reply]
不如叫Jvav好听(bushi
